### PR TITLE
issue-214: added generic icon for new service type

### PR DESCRIPTION
**Pull request by ShaileshParmar-WebDeveloper**
### Describe your changes :

I worked on the generic icon issue because it was not showing any default icon

#
### Type of change :

- [x] Bug fix
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Review by Sachin-chaurasiya — APPROVED** [review]

File: catalog-rest-service/src/main/resources/ui/src/constants/services.const.ts [hunk 1]
```diff
@@ -34,6 +35,7 @@ export const HIVE = hive;
 export const POSTGRES = postgres;
 export const ORACLE = oracle;
 export const SNOWFLAKE = snowflakes;
+export const GENERIC = generic;
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/src/constants/services.const.ts:38]: Use some meaningful name, GENERIC/generic does not clarify the purpose of this icon

Comment (darth-coder00) [reply catalog-rest-service/src/main/resources/ui/src/constants/services.const.ts:38]: One suggestion, you can use something like serviceDefault

**Review by darth-coder00 — CHANGES_REQUESTED** [review]
1 change requested

**Review by darth-coder00 — APPROVED** [review]
LGTM

**Pull request merged**